nchs_mortality/delphi_nchs_mortality/constants.py

```diff
@@ -20,3 +26,4 @@
 ]
 INCIDENCE_BASE = 100000
 GEO_RES = "state"
+NEWLINE = "\n"
```

Comment:
All else equal, I am against macro-ing this.  Based on the use cases in the other file, I would prefer to just put the literal "\n" wherever `NEWLINE` is.

Reply:
[backslash expressions aren't allowed in f-strings, alas](https://www.python.org/dev/peps/pep-0498/#escape-sequences)

Reply:
Can you please add a comment to explain why are aliasing this here then?